build_ffmpeg.sh

```diff
@@ -117,8 +117,17 @@ if [ ! -n "${NUM_JOBS}" ] || [ ! -n "${LEVEL}" ] || [ ! -n "${CPU}" ] ||
     exit 1
 fi
 
-# TOOLCHAIN
+# Determine the output directory. Put PIE executables in their own seperate path.
+# Also set whethere --enable-pic is passed to FFmpeg's configuration.
```

Comment:
whethere->whether.
what is --enable-pic? I guess a PIE-only argument that we need?